src/main/java/FC/Card.java

```diff
@@ -45,4 +45,9 @@ public boolean equals(Object obj) {
         }
 
     }
+
```

Comment:
added toString so JList (for delete card) displays term names not object addresses

Reply:
good call